lib/internal/modules/esm/resolve.js

```diff
@@ -912,7 +922,7 @@ function resolveAsCommonJS(specifier, parentURL) {
   }
 }
 
-function defaultResolve(specifier, context = {}, defaultResolveUnused) {
+async function defaultResolve(specifier, context = {}, defaultResolveUnused) {
   let { parentURL, conditions } = context;
   if (parentURL && policy?.manifest) {
```

Comment:
Am I right in reading this that effectively policy bypasses the checks below?

Reply:
yes, this is intentional by design. policies are the authority on resolution when used, they can defer to normal rules by using a resolution target of boolean `true`.

Reply:
Makes sense.